linkerd/metrics/src/serve.rs

```diff
@@ -33,22 +33,22 @@ impl<M> Serve<M> {
 }
 
 impl<M: FmtMetrics> Serve<M> {
-    pub fn serve<B>(&self, req: http::Request<B>) -> std::io::Result<http::Response<Body>> {
+    pub fn serve<B>(&self, req: http::Request<B>) -> std::io::Result<http::Response<BoxBody>> {
```

Comment:
:point_up: see places like this as well, where we move interfaces towards `BoxBody`.